src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -420,6 +439,10 @@ Report(
     ReportPrometheusMetrics(summary.metrics.front());
   }
 
+  if (summary.overhead_pct > overhead_pct_threshold) {
```

Comment:
What is summary in this case? Is it from one window? Or average across the stable windows?

Reply:
This is the average over the stable windows.